### Hi there ðŸ‘‹

âš¡ Fun fact: My background is in bioinformatics and computational biology.  


### What I do
I currently work at Colorado State University as a researcher in the [Erin Osborne Nishimura laboratory](onishlab.colostate.edu). 

I am also the course coordinator for Quantitative Cell and Molecular Biology, an introductory course on bioinformatics subjects and techniques geared toward first year PhD students.

### Interesting projects

#### Interactive MA-plot and viewer for RNA-seq data

<a href="https://github.com/meekrob/dash-ma-plot"><img alt="Screenshot demo" src="../../../../meekrob/dash-ma-plot/blob/main/screenshots/MA-dash-10fps-50pc.gif" width=448 height=195></a>


#### ELT-2 ChIP-seq and RNA-seq study

Meta-analysis of transcription factor binding sites in *C. elegans*: [https://github.com/meekrob/ELT2-metanalysis](https://github.com/meekrob/ELT2-metanalysis)



#### Animation

Python code to convert *C. elegans* development nuclei-tracking data into Blender models with animation.

<a href="https://github.com/meekrob/c-blenderi"><img alt="Screenshot mid-video" src="../../../../meekrob/c-blenderi/blob/master/screenshots/youtube_metaballs_50pc_10s.gif" width=328 height=195></a>


## Selected Publications

<span class="mark">1. Williams RTP, King DC, Mastroianni IR, Hill JL, Apenes NW, Ramirez G, Miner EC, Moore A, Coleman K, Nishimura EO. *Transcriptome profiling of the Caenorhabditis elegans intestine reveals that ELT-2 negatively and positively regulates intestinal gene expression within the context of a gene regulatory network.* **Genetics**. 2023 Aug 9;224(4):iyad088. doi: 10.1093/genetics/iyad088. PMID: 37183501; PMCID: PMC10411582.</span>

<span class="mark">2. Herman JA, Romain RR, Hoellerbauer P, Shirnekhi
HK, King DC, DeLuca KF, Osborne Nishimura E, Paddison PJ, DeLuca JG.
*Hyper-active RAS/MAPK introduces cancer-specific mitotic
vulnerabilities.* **Proc Natl Acad Sci** U S A. 2022 Oct
11;119(41):e2208255119. doi: 10.1073/pnas.2208255119. Epub 2022 Oct 3.
PMID: 36191188; PMCID: PMC9565228.</span>

<span class="mark">3. Cascarina SM, King DC, Osborne Nishimura E, Ross
ED. *LCD-Composer: an intuitive, composition-centric method enabling the
identification and detailed functional mapping of low-complexity
domains.* **NAR Genom Bioinform.** 2021 May 26;3(2):lqab048. doi:
10.1093/nargab/lqab048. PMID: 34056598; PMCID: PMC8153834.</span>

<span class="mark">4. Kapuscinski ML, Bergren NA, Russell BJ, Lee JS,
Borland EM, Hartman DA, King DC, Hughes HR, Burkhalter KL, Kading RC,
Stenglein MD. *Genomic characterization of 99 viruses from the
bunyavirus families Nairoviridae, Peribunyaviridae, and Phenuiviridae,
including 35 previously unsequenced viruses.* **PLoS Pathog.** 2021 Mar
1;17(3):e1009315. doi: 10.1371/journal.ppat.1009315. PMID: 33647063;
PMCID: PMC7951987.</span>

<span class="mark">5. Cheng Y, Wu W, Kumar SA, Yu D, Deng W, Tripic T,
King DC, Chen KB, Zhang Y, Drautz D, Giardine B, Schuster SC, Miller W,
Chiaromonte F, Zhang Y, Blobel GA, Weiss MJ, Hardison RC. *Erythroid
GATA1 function revealed by genome-wide analysis of transcription factor
occupancy, histone modifications, and mRNA expression.* **Genome Res**.
2009 Dec;19(12):2172-84. doi: 10.1101/gr.098921.109. Epub 2009 Nov 3.
PMID: 19887574; PMCID: PMC2792182.</span>

<span class="mark">7. Zhang Y, Wu W, Cheng Y, King DC, Harris RS, Taylor
J, Chiaromonte F, Hardison RC. *Primary sequence and epigenetic
determinants of in vivo occupancy of genomic DNA by GATA1*. **Nucleic
Acids Res**. 2009 Nov;37(21):7024-38. doi: 10.1093/nar/gkp747. PMID:
19767611; PMCID: PMC2790884.</span>

<span class="mark">8. Cheng Y, King DC, Dore LC, Zhang X, Zhou Y, Zhang
Y, Dorman C, Abebe D, Kumar SA, Chiaromonte F, Miller W, Green RD, Weiss
MJ, Hardison RC. *Transcriptional enhancement by GATA1-occupied DNA
segments is strongly associated with evolutionary constraint on the
binding site motif.* **Genome Res**. 2008 Dec;18(12):1896-905. doi:
10.1101/gr.083089.108. Epub 2008 Sep 25. PMID: 18818370; PMCID:
PMC2593580.</span>

<span class="mark">9. Miller W, Rosenbloom K, Hardison RC, Hou M, Taylor
J, Raney B, Burhans R, King DC, Baertsch R, Blankenberg D, Kosakovsky
Pond SL, Nekrutenko A, Giardine B, Harris RS, Tyekucheva S, Diekhans M,
Pringle TH, Murphy WJ, Lesk A, Weinstock GM, Lindblad-Toh K, Gibbs RA,
Lander ES, Siepel A, Haussler D, Kent WJ. *28-way vertebrate alignment
and conservation track in the UCSC Genome Browser.* **Genome Res.** 2007
Dec;17(12):1797-808. doi: 10.1101/gr.6761107. Epub 2007 Nov 5. PMID:
17984227; PMCID: PMC2099589.</span>

<span class="mark">10. ENCODE Project Consortium; *Identification and
analysis of functional elements in 1% of the human genome by the ENCODE
pilot project.* **Nature.** 2007 Jun 14;447(7146):799-816. doi:
10.1038/nature05874. PMID: 17571346; PMCID: PMC2212820.</span>

<span class="mark">11. King DC, Taylor J, Zhang Y, Cheng Y, Lawson HA,
Martin J; ENCODE groups for Transcriptional Regulation and Multispecies
Sequence Analysis; Chiaromonte F, Miller W, Hardison RC. *Finding
cis-regulatory elements using comparative genomics: some lessons from
ENCODE data.* **Genome Res.** 2007 Jun;17(6):775-86. doi:
10.1101/gr.5592107. PMID: 17567996; PMCID: PMC1891337.</span>

<span class="mark">12. Taylor J, Tyekucheva S, King DC, Hardison RC,
Miller W, Chiaromonte F. *ESPERR: learning strong and weak signals in
genomic sequence alignments to identify functional elements.* **Genome
Res.** 2006 Dec;16(12):1596-604. doi: 10.1101/gr.4537706. Epub 2006 Oct
19. PMID: 17053093; PMCID: PMC1665643.</span>

<span class="mark">13. Wang H, Zhang Y, Cheng Y, Zhou Y, King DC, Taylor
J, Chiaromonte F, Kasturi J, Petrykowska H, Gibb B, Dorman C, Miller W,
Dore LC, Welch J, Weiss MJ, Hardison RC. *Experimental validation of
predicted mammalian erythroid cis-regulatory modules.* **Genome Res.**
2006 Dec;16(12):1480-92. doi: 10.1101/gr.5353806. Epub 2006 Oct 12.
PMID: 17038566; PMCID: PMC1665632.</span>

<span class="mark">14. King DC, Taylor J, Elnitski L, Chiaromonte F,
Miller W, Hardison RC. *Evaluation of regulatory potential and
conservation scores for detecting cis-regulatory modules in aligned
mammalian genome sequences.* **Genome Res.** 2005 Aug;15(8):1051-60.
doi: 10.1101/gr.3642605. Epub 2005 Jul 15. PMID: 16024817; PMCID:
PMC1182217.</span>

<span class="mark">15. International Chicken Genome Sequencing
Consortium. *Sequence and comparative analysis of the chicken genome
provide unique perspectives on vertebrate evolution.* **Nature**. 2004
Dec 9;432(7018):695-716. doi: 10.1038/nature03154. Erratum in: Nature.
2005 Feb 17;433(7027):777. PMID: 15592404.</span>

<span class="mark">16. Putta S, Smith JJ, Walker JA, Rondet M, Weisrock
DW, Monaghan J, Samuels AK, Kump K, King DC, Maness NJ, Habermann B,
Tanaka E, Bryant SV, Gardiner DM, Parichy DM, Voss SR. *From biomedicine
to natural history research: EST resources for ambystomatid
salamanders.* **BMC Genomics**. 2004 Aug 13;5(1):54. doi:
10.1186/1471-2164-5-54. PMID: 15310388; PMCID: PMC509418.</span>

<span class="mark">Shah NH, King DC, Shah PN, Fedoroff NV. *A tool-kit
for cDNA microarray and promoter analysis.* **Bioinformatics**. 2003 Sep
22;19(14):1846-8. doi: 10.1093/bioinformatics/btg253. PMID:
14512358.</span>

